core/domain/classifier_services.py

```diff
@@ -127,6 +127,75 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+def handle_classifier_training_job_creation(exploration, state_names):
```

Comment:
@seanlip Yep. We can make them the same function, but the function gets too large with lots of code. So, I'd suggest making create_multi_classifier_training_jobs into _create_multi_classifier_training_jobs() and call the handle_classifier_training_job_creation function - create_multi_classifier_training_jobs. Is that fine?